personalization-service/src/main/java/de/muenchen/dbs/personalization/checklist/ChecklistService.java

```diff
@@ -0,0 +1,73 @@
+package de.muenchen.dbs.personalization.checklist;
+
+import static de.muenchen.dbs.personalization.common.ExceptionMessageConstants.MSG_NOT_FOUND;
+
+import de.muenchen.dbs.personalization.checklist.domain.Checklist;
+import de.muenchen.dbs.personalization.checklist.domain.ChecklistItem;
+import de.muenchen.dbs.personalization.common.NotFoundException;
+import de.muenchen.dbs.personalization.security.Authorities;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class ChecklistService {
+
+    private final ChecklistRepository checklistRepository;
+
+    @PreAuthorize(Authorities.CHECKLIST_CREATE)
+    public Checklist createChecklist(final String userId, final String title, final List<String> serviceIds) {
+        log.debug("Create Checklist for {} with {}", userId, serviceIds);
+        final Checklist createdChecklist = new Checklist();
+        createdChecklist.setLhmExtId(userId);
+        createdChecklist.setLastUpdate(ZonedDateTime.now());
+        createdChecklist.setTitle(title);
+        // TODO Get service details from ServiceApi
```

Comment:
I don't think we need this TODO.

I would prefer loading the Checklistitem information when we read the checklist. What do you think?